src/core/include/openvino/core/except.hpp

```diff
@@ -25,7 +25,7 @@ class OPENVINO_API Exception : public std::runtime_error {
     OPENVINO_DEPRECATED("This constructor is deprecated and will be removed, please use OPENVINO_THROW instead")
     explicit Exception(const std::string& what_arg);
 
-    [[noreturn]] static void create(const CheckLocInfo& check_loc_info, const std::string& explanation);
+    [[noreturn]] static void create(const char* file, int line, const std::string& explanation);
```

Comment:
Looks like `CheckLocInfo` not used anymore

Reply:
It still used in the `make_what`.
As it is used in .cpp impact on bin-size is minimal.
It can be candidate for removal, but maybe some additional compilation issues can pop-up like now for vpux plugin.
First fix all issues with this changes then it can be fine tune to remove this struct.